LiskHQ/lisk-hub
# Review and refactor the actions, reducers and middleware for transaction - Closes #2176

### What issue have I solved?

-- #2176 

### How have I implemented/fixed it?

- [x] Improve naming for transactions actions functions.
- [x] Replace promises to use async/await
- [x] Add comments to understand better the functions
- [x] Add TODO comments in the reducer and actions file for remove once get single transaction data, create tx and broadcast tx code be part of utils.
- [x] Update all unit tests file and components to use the new names

### How has this been tested?

1. Login into the app.
2. Check dashboard data load properly (recent tx).
3. Go to wallet see all data load properly (all tx, incoming and outgoing) without errors
4. Check any tx and see the details of the tx

In summary all the app should continue working as before without any problem


### Review checklist
- The PR follows our [Test guide](/LiskHQ/lisk-hub/blob/development/docs/TEST_GUIDE.md)
- The PR follows our [CSS guide](/LiskHQ/lisk-hub/blob/development/docs/CSS_GUIDE.md)
